corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -347,7 +348,7 @@ def get_choices_for_known_values(self, values, user):
     @staticmethod
     def get_choices_from_es_query(group_es):
         return [Choice(group_id, name)
-                for group_id, name in group_es.values_list('_id', 'name')]
+                for group_id, name in values_list(group_es.scroll(), '_id', 'name')]
```

Comment:
Can you use `scroll_ids` here instead? This will fetch the whole doc from ES instead of just the `id`.

Reply:
Scroll_ids will return a list of ids right? But here we need to fetch `'_id', 'name'` from the docs.
Does that sound correct?
@proteusvacuum

Reply:
ah, missed that, sorry!

In that case, I think you also want to only include the `_id` and `name` fields to be fetched from ES. You can see how the `values_list` method does that here:
https://github.com/dimagi/commcare-hq/blob/13cbf3fd7bda47d5aff6ac3357d5d81bc5a2fe8b/corehq/apps/es/es_query.py#L469-L470

You should be able to use the `source` method to only select the pertinent fields:
```py
values_list(group_es.source(include=['_id', 'name']).scroll(), '_id', 'name')
```

or ideally you could modify the utility methods in `es_query.py` to be a bit smarter - something like:

```py
def values_list(self, *fields, **kwargs):
    if kwargs.pop('scroll', False):
        hits = self.fields(fields).scroll()
    else:
        hits = self.fields(fields).run().hits,
    return values_list(hits, *fields, **kwargs)
```